e2e/cypress/integration/system_console/site_configuration/announcement_banner_spec.js

```diff
@@ -102,14 +103,13 @@ describe('Announcement Banner', () => {
             contains('a', bannerEmbedLink).
             should('have.attr', 'href', bannerEmbedLink);
 
-        // * Verify only the banner text's first part is visible
-        // and check the complete text length spans more than viewport width (also is hidden as per above)
+        // * Verify only the banner text's first part is visible by ensuring the first link
```

Comment:
I had to change this test because the paragraph is now what is truncating the text, not the element that contains the paragraph, so the paragraph is now constrained by the screen while its contents are wider. Since I don't think we can measure arbitrary element contents, I think it works just as well if we check that a child at the start is visible and a child at the end isn't.